KS.Fiks.IO.Arkiv.Client/Schema/sok.xsd

```diff
@@ -231,5 +232,125 @@
             <xs:enumeration value="dokumentobjekt"/>
         </xs:restriction>
     </xs:simpleType>
+
+    <xs:complexType name="sortering">
+        <xs:sequence>
+            <xs:element name="felt" type="sorteringFelt" minOccurs="1" maxOccurs="1"/>
+            <xs:element name="type" type="sorteringType" minOccurs="1" maxOccurs="1"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:simpleType name="sorteringType">
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="synkende"/>
+            <xs:enumeration value="stigende"/>
+        </xs:restriction>
+    </xs:simpleType>
+
+    <xs:simpleType name="sorteringFelt">
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="mappe.eksternId"/>
```

Comment:
referanseEksternNokkel?
Denne er et objekt med to felt. Sortering kan være umulig via SQL.